paketosbom/sbom.go

```diff
@@ -76,4 +76,5 @@ const (
 	BLAKE2B512 algorithm = "BLAKE2b-512"
 	BLAKE3     algorithm = "BLAKE3"
 	MD5        algorithm = "MD5"
+	UNKNOWN    algorithm = "UNKNOWN"
```

Comment:
@paketo-buildpacks/tooling-maintainers I thought the addition of the `UNKNOWN` algorithm would make it clearer in the output when something has gone wrong since the `GenerateBillOfMaterials` function does not return an error of any kind. I can remove this if you think it's unnecessary though.